docs/reference/esql/functions/pow.asciidoc

```diff
@@ -16,62 +17,6 @@ include::{esql-specs}/math.csv-spec[tag=powDI]
 include::{esql-specs}/math.csv-spec[tag=powDI-result]
 |===
 
-[discrete]
-==== Type rules
-
-The type of the returned value is determined by the types of the base and exponent.
-The following rules are applied to determine the result type:
-
-* If either of the base or exponent are of a floating point type, the result will be a double
-* Otherwise, if either the base or the exponent are 64-bit (long or unsigned long), the result will be a long
-* Otherwise, the result will be a 32-bit integer (this covers all other numeric types, including int, short and byte)
-
-For example, using simple integers as arguments will lead to an integer result:
-
-[source.merge.styled,esql]
-----
-include::{esql-specs}/math.csv-spec[tag=powII]
-----
-[%header.monospaced.styled,format=dsv,separator=|]
-|===
-include::{esql-specs}/math.csv-spec[tag=powII-result]
-|===
-
-NOTE: The actual power function is performed using double precision values for all cases.
-This means that for very large non-floating point values there is a small chance that the
-operation can lead to slightly different answers than expected.
-However, a more likely outcome of very large non-floating point values is numerical overflow.
-
-[discrete]
-==== Arithmetic errors
```

Comment:
Is it no longer possible to generate an arithmetic error? This section could remain, but with a new example. On the other hand perhaps this is general to all numerical functions, and should be in a generic location.

Reply:
Yeah, it just behaves like the rest of the numeric functions right now.  Also, there are just too many ways to get an error, and too many weird edge cases.  I didn't want to copy all the behavior from the java docs.